pattern_manager/PatternParent parent
pattern_manager/Point[] points
---
bool success